src/ucp/wireup/wireup_cm.c

```diff
@@ -431,9 +448,14 @@ static unsigned ucp_cm_client_uct_connect_progress(void *arg)
 
     can_fallback = (ep_init_flags != UCP_EP_INIT_CREATE_AM_LANE_ONLY) ||
                    (ucp_cm_client_get_next_cm_idx(ep) != UCP_NULL_RESOURCE);
+
+    if (ep_init_flags & UCP_EP_INIT_CREATE_AM_LANE_ONLY) {
+        address_mode = UCP_WIREUP_SA_DATA_AM_ONLY;
+    }
     status       = ucp_cm_ep_priv_data_pack(ep, &tl_bitmap, dev_index,
                                             can_fallback,
-                                            &priv_data, &priv_data_length);
+                                            &priv_data, &priv_data_length, 1,
```

Comment:
does it mean we pack client_id even if not passed by the user?

Reply:
So one option is to set client_id to the default value, say `UINT64_MAX` and always pass it, or to reserve this value and check for it, then don't send client id, or to add other field to wireup_ep - client_id_set. What do you prefer?

Reply:
As discussed default client_id value on both sides is 0 and won't be send